README.md

```diff
@@ -2,7 +2,17 @@
 
 ## Purpose
 
-_... help fill this in!_
+* Define and maintain security policies and procedures for:
+  - the core Node.js project
+  - other projects maintained by the Node.js Foundation technical group
+* Define and maintain policies and procedures for the coordination of security concerns within the external Node.js open source ecosystem
+* Maintain and make available data on disclosed security vulnerabilities in:
+  - the core Node.js project
+  - other projects maintained by the Node.js Foundation technical group
+  - the external Node.js open source ecosystem
+* Promote improvement of security practices within the Node.js ecosystem
+* Recommend security improvements for the core Node.js project
+* Facilitate and promote the expansion of a healthy security service and product provider ecosystem
```

Comment:
concrete example: https://medium.com/lift-security/were-donating-the-node-security-project-to-the-node-js-foundation-1776e351394a#.hajz1xcy7